openapi/calendar/calendar.yaml

```diff
@@ -0,0 +1,1535 @@
+openapi: 3.0.1
+info:
+  title: CalendarHero API
+  termsOfService: 'https://calendarhero.com/terms-of-use'
+  contact:
+    name: CalendarHero
+    url: 'https://faq.calendarhero.com'
+    email: hello@calendarhero.com
+  version: '1.0'
+servers:
+  - url: 'https://prod.apigateway.co/products/calendar'
+    description: Production
+  - url: 'https://demo.apigateway.co/products/calendar'
+    description: Demo
+  - url: '{local}/products/calendar'
+    description: Local
+  - url: 'http://localhost:11001/products/calendar'
+    description: Localhost
+tags: []
+paths:
+  /contact:
+    get:
+      tags:
+        - contact
+      summary: List Contacts
+      operationId: getContact
+      parameters:
+        - name: search
+          in: query
+          description: 'searches for contacts with a specific email (name@domain.com), domain (@domain.com) or name (first last)'
+          schema:
+            type: string
+        - name: filter
+          in: query
+          description: searches for a specific set of contacts. can not be used with search
+          schema:
+            type: string
+            enum:
+              - popular
+              - recent
+              - a
+              - b
+              - c
+              - d
+              - e
+              - f
+              - g
+              - h
+              - i
+              - j
+              - k
+              - l
+              - m
+              - 'n'
+              - o
+              - p
+              - q
+              - r
+              - s
+              - t
+              - u
+              - v
+              - w
+              - x
+              - 'y'
+              - z
```

Comment:
What's this?

Reply:
Is this file just an example of how to add CH documentation?

Reply:
I imported the existing API docs from https://api.calendarhero.com/documentation and then changed the server and auth details. 

To maintain this file going forward we have two options:
1. Improve the existing CH API docs and set up a build process to automatically make the same changes that I did while copying updates to this repo.
2. Use this repo as the source of truth and follow [this guide](https://vendasta.jira.com/wiki/spaces/API/pages/1670709369/Updating+Documentation#Stoplight-Content) to edit the docs using Stoplight Studio.

If we decide to change from a raw proxy to actually rewriting the request and response bodies (so we can filter out some deprecated fields) then option 2 is the way to go.